test/core/runtime/tagged_transaction_queue_test.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime/impl/tagged_transaction_queue_impl.hpp"
+
+#include <gtest/gtest.h>
+#include "core/runtime/runtime_fixture.hpp"
+#include "core/storage/merkle/mock_trie_db.hpp"
+#include "extensions/extension_impl.hpp"
+#include "primitives/impl/scale_codec_impl.hpp"
+#include "runtime/impl/wasm_memory_impl.hpp"
+#include "testutil/outcome.hpp"
+#include "testutil/runtime/wasm_test.hpp"
+
+using namespace testing;
+using kagome::common::Buffer;
+using kagome::extensions::ExtensionImpl;
+using kagome::primitives::Extrinsic;
+using kagome::primitives::ScaleCodecImpl;
+using kagome::runtime::TaggedTransactionQueue;
+using kagome::runtime::TaggedTransactionQueueImpl;
+using kagome::runtime::WasmMemoryImpl;
+using kagome::storage::merkle::MockTrieDb;
+
+class TTQFixture: public RuntimeFixture {
```

Comment:
It is not necessary, but maybe it is more appropriate to use Test as suffix instead of Fixture.